lib/engine/game/g_1860.rb

```diff
@@ -124,6 +138,24 @@ def or_set_finished
         check_new_layer
       end
 
+      def place_home_track(corporation)
+        hex = hex_by_id(corporation.coordinates)
+        tile = hex&.tile
```

Comment:
why do you do &.tile here, line 146 indicates that tile is never nil

Reply:
Yeah, that's inconsistent. Is there ever a case where a hex doesn't have a tile?